src/data/attribute.rs

```diff
@@ -166,7 +166,10 @@ fn process_metadata(s: &Struct, prefix: String) -> Vec<(String, String)> {
             let nested_struct = value.get_struct_value();
             result.extend(process_metadata(nested_struct, current_prefix));
         } else if let Some(v) = json {
-            result.push((current_prefix, serde_json::to_string(&v).unwrap()));
+            result.push((
+                current_prefix,
+                serde_json::to_string(&v).expect("failed to serialize json Value"),
```

Comment:
Would this contain any information about `v`? If not, maybe `.inspect` or otherwise use the error to log?
I'd maybe even go as far as dropping that mapping on the floor, log at error! Or explicitly add a `null` value or... rather than having the whole process crash?

Reply:
In what cases would the serialization fail?

Reply:
Right, given the above, _it can only be_ `String`, `Bool`, `null` or some Number... so I guess it shouldn't ever fail indeed... for as long as that invariant remains true tho...